docs/js/user.js

```diff
@@ -59,9 +60,486 @@ var userAccount = (function(){
+      }
+   }
+   function applyFilters(){
+      if(!loading){
+         let dateFrom = document.getElementById(ids.dateFrom)
+         let dateTo = document.getElementById(ids.dateTo)
+         if(dateFrom.value && dateTo.value){
+            tFrom = Date.parse(dateFrom.value)/1000
+            tTo = Date.parse(dateTo.value)/1000
+            page = 0;
+            updatePageNumber();
+            updateTable();
+         }else{
+            let toastHTML = 'invalid data'
+            M.toast({html: toastHTML,displayLength:2000, classes:"dark-toast"}); 
+         }
+      }
+   }
+   function clearFilters(){
+      if(!loading){
+         document.getElementById(ids.dateFrom).value = '';
+         document.getElementById(ids.dateTo).value = '';
+         page = 0;
+         tFrom = '';
+         tTo = '';
+         updatePageNumber();
+         updateTable();
+      }
+   }
+   function previous(){
+      if(!loading){
+         page--
+         updatePageNumber();
+         updateTable()
+      }
+   }
+   function updatePageNumber(){
+      let previousBt = document.getElementById(ids.page.previous).classList;
+      let nextBt = document.getElementById(ids.page.next).classList;
+      document.getElementById(ids.page.indicator).innerText = (page+1)+'/'+pages
+      
+      if(page>0) previousBt.remove('disabled')
+      else previousBt.add('disabled')
+   
+      if((page+1)<pages) nextBt.remove('disabled')
+      else nextBt.add('disabled')
+   }
+   function next(){
+      if(!loading){
+         page++
+         updatePageNumber();
+         updateTable()
+      }
+   }
+   function updateTable(){
+      showOverlay()
+      let options = `?per_page=${perPage}&page=${page}&from=${tFrom}&to=${tTo}`
+      jsonApi.get(`/investor/${name}/investments${options}`)
+      .then(d=>{
+         if(d.length>0)render(d)
+         else noInvestments()
+         removeOverlay()
+         profitChart.updateDataSet(d)
+      })
+      .catch(er=>{
+         removeOverlay()
+         connectionErrorToast(er);
+      })      
+   }
+   function showOverlay(){
+      loading = true;
+      document.getElementById(ids.table).style.opacity = 0.4
+      document.getElementById(ids.tableOverlay).style.opacity = 1      
+   }
+   function removeOverlay(){ 
+      let overlay = document.getElementById(ids.tableOverlay)
+      document.getElementById(ids.table).style.opacity = 1
+      overlay.classList.add('pulse-out')
+      setTimeout(_=>{
+         loading = false;
+         overlay.style.opacity = 0
+         overlay.classList.remove('pulse-out')
+      },400)
+   }
+   function noInvestments(){
+      document.getElementById(ids.table).innerHTML = 
+      `<h5 class="grey-text">No investments found</h5>`
+      document.getElementById(ids.page.next).classList.add('disabled')
+   }
+   function render(data){
+      console.log(data)
+      let html = `
+      <thead>
+        <tr>
+            <th>Post url</th>
```

Comment:
Let's clean up the table headers a little. "Post url" should maybe just be "post", and "In" (capital i) should just be "in" (lowercase i)